### ISSUE TITLE
Remove drive direction toggle

### ISSUE THREAD

**Issue by rhauch** [frc-4931/2016-Robot#77]
Currently the driver can pull the trigger to reverse the direction of the robot. This is confusing and causes problems with new drivers, so we should remove this.

**Comment by agausmann** [comment]
I agree, but may be a potentially useful feature. Maybe we could move it to a button that is less easily bumped?
